Frameworks/Foundation/NSFileManager.mm

```diff
@@ -436,36 +436,48 @@ - (BOOL)copyItemAtPath:(id)srcPath toPath:(id)destPath error:(NSError**)error {
 
     TraceVerbose(TAG, L"Done copying");
 
-    return TRUE;
+    return YES;
 }
 
 /**
- @Status Stub
+ @Status Caveat
+ @Notes does not support file manager move delegates for shouldMoveItem or shouldProceedAfterError.
 */
 - (BOOL)moveItemAtURL:(NSURL*)srcURL toURL:(NSURL*)dstURL error:(NSError**)error {
-    UNIMPLEMENTED();
-
-    return NO;
+    return [self moveItemAtPath:[srcURL path] toPath:[dstURL path] error:error];
```

Comment:
I was letting it fall down to the fact if it's not fileURL then we throw exception. looks like reference platform returns an error with the exception.

---
In reply to: [71392720](https://github.com/Microsoft/WinObjC/pull/669#discussion_r71392720) [](ancestors = 71392720)